keyrings/live/blackbox-admins.txt

```diff
@@ -0,0 +1 @@
+beamalsky@datamade.us
```

Comment:
Could add me and Forest as admins?

Reply:
Use my Gmail: reginafcompton@gmail.com

Reply:
@fgregg what email do you use for blackbox?